srcpkgs/polari/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'polari'
 pkgname=polari
-version=45.0
+version=46.0
 revision=1
 build_style=meson
 build_helper="gir"
 hostmakedepends="pkg-config itstool gettext glib-devel gtk-update-icon-cache
  desktop-file-utils"
-makedepends="gjs-devel telepathy-logger-devel telepathy-glib-devel"
-depends="telepathy-logger telepathy-glib libsecret gtk4 libadwaita
+makedepends="gjs-devel telepathy-glib-devel tracker-devel"
+depends="tracker telepathy-glib libsecret gtk4 libadwaita
```

Comment:
Remove `tracker` from here. Only libtracker is required which already gets pulled in due to shlibs.